lib/azure/armrest/armrest_manager.rb

```diff
@@ -167,6 +170,8 @@ def initialize(options = {})
 
         # Base URL used for REST calls. Modify within method calls as needed.
         @base_url = Azure::Armrest::RESOURCE
+
```

Comment:
@bzwei Possibly, but let's save that for another PR. That's an easy refactoring.